README.adoc

```diff
@@ -50,21 +50,23 @@ You can download/copy the three files 'checksums.txt.pem', 'checksums.txt.sig',
 Once you have the three files locally, you can execute the following command
 
 ```
-COSIGN_EXPERIMENTAL=1 cosign verify-blob --cert checksums.txt.pem --signature checksums.txt.sig checksums.txt
+cosign verify-blob --certificate-identity-regexp "https://github.com/updatecli/updatecli" --certificate-oidc-issuer 'https://token.actions.githubusercontent.com' --cert https://github.com/updatecli/updatecli/releases/download/v0.63.0/checksums.txt.pem --signature https://github.com/updatecli/updatecli/releases/download/v0.63.0/checksums.txt.sig checksums.txt
```

Comment:
I'll need to add an updatecli rule to update this url as well https://github.com/updatecli/updatecli/blob/491860d624997925042f217f09a6cc86ac43d8b2/updatecli/updatecli.d/updatecli.yaml#L57

Reply:
But today the UX makes it easier to do in two times